bot/cogs/error_handler.py

```diff
@@ -49,10 +51,14 @@ def __init__(self, bot: Bot):
             return
 
         # Try to look for a tag with the command's name if the command isn't found.
-        if isinstance(e, errors.CommandNotFound) and not hasattr(ctx, "invoked_from_error_handler"):
-            if ctx.channel.id != Channels.verification:
+        if isinstance(e, errors.CommandNotFound):
+            if (
+                    not await self.try_silence(ctx)
+                    and not hasattr(ctx, "invoked_from_error_handler")
+                    and ctx.channel.id != Channels.verification
+            ):
                 await self.try_get_tag(ctx)
-                return  # Exit early to avoid logging.
+            return  # Exit early to avoid logging.
```

Comment:
This is wrong. Logging should only be avoided if a tag or the hush command succeeds.